Assets/Unity3DTiles/Unity3DTile.cs

```diff
@@ -179,10 +179,8 @@ public Unity3DTile(Unity3DTileset tileset, string basePath, Schema.Tile tile, Un
 
             // TODO: Consider using a double percision Matrix library for doing 3d tiles root transform calculations
             // Set the local transform for this tile, default to identity matrix
-            this.transform = this.tile.UnityTransform();
-            
-            var parentTransform = (parent != null) ? parent.computedTransform : Matrix4x4.identity;
-            this.computedTransform = parentTransform * this.transform;
+            this.computedTransform = this.tileset.TilesetOptions.Transform;
```

Comment:
I had changed the parent so that transforms were all handled at the tileset level. `this.transform` is only used for getting `this.computedTransform`, which is now known from `this.tileset.TilesetOptions.Transform`. I'm refactoring to make this more readable and correctly set `this.transform` regardless

Reply:
Ah I see now, tiles can have optional transforms to their parent tile.

Reply:
So now, each tile's `transform` is set relative to it's parent tile, and `computedTransform` includes all transforms up the tree plus the tileset transform. Transforms at the tileset level can then go away.